code-challenges.js

```diff
@@ -39,16 +51,55 @@ var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var newVowelTester1 = vowelTester1.split("")
+var vowels = ["a","e","i","o","u"]
+
+var vowelTester2 = "throw"
 // Expected output: 3
+var newVowelTester2 = vowelTester2.split("")
+
+function findCommonElement1(array1, array2) {
 
+    // Loop for array1
+    for(let i = 0; i < array1.length; i++) {
 
+        // Loop for array2
+        for(let j = 0; j < array2.length; j++) {
 
+            // Compare the element of each and
+            // every element from both of the
+            // arrays
+            if(array1[i] == array2[j]) {
+                return i;
+            }
+        }
+    }
+    return false;
+}
 
+console.log(findCommonElement1(newVowelTester1,vowels))
+console.log(findCommonElement1(newVowelTester2,vowels))
 
 
 // --------------------5) Create a function that takes three arguments - two numbers and a mathematical operation (+, -, *, /) and performs the given calculation. If the input tries to divide by 0, return: "Can't divide by 0!"
 
+const calculator = (num1,operator, num2) => {
+	if (operator == "/" && (num1==0 || num2==0)){
```

Comment:
by this logic, i think only num2 would beed to === 0 for the correct return to run?